.browserslistrc

```diff
@@ -9,60 +9,98 @@ node 14
 # `npx browserslist --mobile-to-desktop "> 0.5%, last 2 versions, Firefox ESR, not dead, not IE 11"` when the last major is released.
 # On update check all #stable-snapshot markers
 [stable]
-and_chr 85
-and_ff 80
+and_chr 92
+and_chr 91
```

Comment:
Let's limit these to the lowest version per entry. Applies to other browsers as well.

Reply:
Updated